sdformat_urdf/src/sdformat_urdf.cpp

```diff
@@ -493,12 +493,16 @@ sdformat_urdf::convert_joint(const sdf::Joint & sdf_joint, sdf::Errors & errors)
     case sdf::JointType::PRISMATIC:
       urdf_joint->type = urdf::Joint::PRISMATIC;
       break;
+    case sdf::JointType::UNIVERSAL:
+      urdf_joint->type = urdf::Joint::FLOATING;
+      break;
+    case sdf::JointType::BALL:
+      urdf_joint->type = urdf::Joint::FLOATING;
+      break;
```

Comment:
Nit, I recommend using `switch/case` fall-through to de-duplicate the code assigning `urdf_joint_->type`.

```suggestion
    case sdf::JointType::UNIVERSAL:  // Fall through
    case sdf::JointType::BALL:
      urdf_joint->type = urdf::Joint::FLOATING;
      break;
```